Simulation Code/Base/Base.ino

```diff
@@ -106,7 +113,6 @@ void loop()
   unsigned long currentTime, elapsedTime, currentPumpTime, elapsedPumpTime; // Variables to store the current time and elapsed time
 
   checkWaterLevel(waterLevelA, waterLevelB, waterLevelC); // Check the current water level
```

Comment:
The point of the struct is to have it passed into functions all at once. For example, checkWaterLevel(WaterLevelStruct);